improver/cli/__init__.py

```diff
@@ -244,13 +246,14 @@ def constrained_inputcubelist_converter(to_convert):
 
         cubelist = maybe_coerce_with(load_cubelist, to_convert)
 
-        return CubeList(
-            cubelist.extract(
+        result = CubeList([])
+        for constr in constraints:
+            item = cubelist.extract(
                 Constraint(cube_func=constr) if callable(constr) else constr,
-                strict=True,
+                strict=strict,
             )
-            for constr in constraints
-        )
+            result.append(item) if strict else result.extend(item)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expression "result.append(item) if strict else result.extend(item)" is assigned to nothing](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5439763)